tox.ini

```diff
@@ -15,9 +15,9 @@ passenv = *
 
 [testenv:js]
 commands =
-    npm install --no-bin-links --prefix {toxinidir}
```

Comment:
Can you leave this line as is? It works around a shared folder issue we saw regarding symlinks on OS X